### PR TITLE
PR for trello case with this id - ZVA1zA8S

### PR DESCRIPTION
**Pull request by melody2020nerd**
cut and pasted the bootstrap folder to the main/root path

### CODE REVIEW

File: lapma/db.php [hunk 1]
```diff
@@ -3,7 +3,7 @@
 $server = "localhost";
 $user = "root";
 $password = "";
-$database = "lapma";
+$database = "yeolomco_lapma";
```
Comment (yeolomlead) [inline lapma/db.php:6]: this is local, did you do this for the Script?

Comment (melody2020nerd) [reply lapma/db.php:6]: yes I changed the db name

Comment (yeolomlead) [reply lapma/db.php:6]: noted

**Review by yeolomlead — APPROVED** [review]
The PR is good to go.

**Pull request merged**